oaknational/Oak-Web-Application
# [CUR-287] updated partner image to square and added responsive sizing to partner card

## Description

- Changes the partner image from circular to square
- Updated flex direction from row to column for mobile view
- Updated spacing underneath the partner card

## How to test

1. Go to https://deploy-preview-1945--oak-web-application.netlify.thenational.academy
2. Click on any subject phase combination and scroll to the partner information
3. You should see an updated image for the partner and spacing 

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2023-09-27 at 17 21 30](https://github.com/oaknational/Oak-Web-Application/assets/32605982/ce5dfa9e-3c34-4f8a-a657-c18b8ceaa6b6)

How it should now look:
{screenshots
![Screenshot 2023-09-27 at 17 20 34](https://github.com/oaknational/Oak-Web-Application/assets/32605982/15c683d5-6d2b-4cf6-87d8-6f168cff0938)

## Checklist
